source/base/timer.cc

```diff
@@ -736,7 +736,7 @@ TimerOutput::print_summary() const
                  << "| Total CPU/wall time elapsed since start     "
                  << extra_space << "|" << std::setw(10) << std::setprecision(3)
                  << std::right << total_cpu_time << "s |            |"
-                 << total_wall_time << "s |            |"
+                 << std::setw(10) << total_wall_time << "s |            |"
```

Comment:
Let's keep this out of this patch for now. It's unrelated to the typo fixes. Feel free to submit it as a separate patch if you think that it was wrong before.